charts/portworx/templates/serviceaccount-hook.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
   name: {{ template "px.hookServiceAccount" . }}
   namespace: kube-system
   annotations:
-    "helm.sh/hook-delete-policy": before-hook-creation
-    "helm.sh/hook": "pre-delete,post-delete,post-install"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    "helm.sh/hook": "post-install,pre-upgrade,pre-delete"
```

Comment:
I think this should be post-delete, to let all resources be deleted before deleting the hook service account. I think I copied it from daemonset which has both pre-delete and post-delete, looks like it was a bug in daemonset chart.

Reply:
The `helm.sh/hook` annotation defines what hook this service account is used for. As we added a pre-delete hook for cluster deletion, we need to use pre-delete and not post-delete. We don't need a post-delete hook as of now.